pkg/filter/filter.go

```diff
@@ -70,16 +74,24 @@ func calibrateFilters(j *ffuf.Job, responses []ffuf.Response) {
 			// Only add if we have an actual word length of response
 			wordCalib = append(wordCalib, strconv.FormatInt(r.ContentWords, 10))
 		}
+		if r.ContentLines > 1 {
+			// Only add if we have an actual word length of response
+			lineCalib = append(lineCalib, strconv.FormatInt(r.ContentLines, 10))
+		}
 	}
 
 	//Remove duplicates
 	sizeCalib = ffuf.UniqStringSlice(sizeCalib)
 	wordCalib = ffuf.UniqStringSlice(wordCalib)
+	lineCalib = ffuf.UniqStringSlice(lineCalib)
 
 	if len(sizeCalib) > 0 {
 		AddFilter(j.Config, "size", strings.Join(sizeCalib, ","))
 	}
 	if len(wordCalib) > 0 {
 		AddFilter(j.Config, "word", strings.Join(wordCalib, ","))
 	}
+	if len(lineCalib) > 0 {
+		AddFilter(j.Config, "line", strings.Join(lineCalib, ","))
```

Comment:
I think that the concept of response line count is too broad to add to the auto-calibration. There's a good chance that all the `404`'s have a single line response, while all the actual legit responses we'd like to catch have a single line too.

Reply:
Do you want to totally remove the concept of autocalibration on the line count ? :)

Reply:
I copied all the code for the words like an idiot without really thinking :)

Just to have this feature which is pretty much the same than the word count one

Reply:
Yeah, I would like the line count to be removed from the autocalibration. I think it will contribute to a whole lot of misses, as it would filter everything based on the line count regardless of the return code after that.

Reply:
all right